### PR TITLE
Making callbacks agile. Fixing seek in ebrstroragefile.

### PR DESCRIPTION
**Pull request by pradipd**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1344)

### CODE REVIEW

**Comment by pradipd** [comment]
@bbowman is added to the review. #Closed

**Comment by bbowman** [comment]
:shipit:

**Comment by DHowett-MSFT** [comment]
:shipit:

File: Frameworks/include/WRLHelpers.h [hunk 1]
```diff
@@ -156,7 +156,7 @@ HRESULT ForEach(ABI::Windows::Foundation::Collections::IMap<K, V>* map, Q&& func
 }
 
 #pragma region Async Helpers
-struct ActionCallback : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::ClassicCom>,
+struct ActionCallback : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::WinRtClassicComMix>,
```
Comment (pradipd) [inline Frameworks/include/WRLHelpers.h]: No worries.  Thanks!

---
In reply to: [87461400](https://github.com/Microsoft/WinObjC/pull/1344#discussion_r87461400) [](ancestors = 87461400)

**Pull request merged**